docs/content/tutorials/database-ingress/README.md

```diff
@@ -0,0 +1,46 @@
+# Database Load Balancing
+
+NGINX Ingress Controller supports load balancing arbitrary TCP and UDP services using a Custom Resource Definition called a TransportServer.
+
+## Usage
+
+This directory contains a sample using redis; this can easily be adapted to other services.
+
+## Installing NGINX Ingress Controller
+
+This tutorial uses Helm to deploy the Ingress Controller. The Image for the Ingress Controller is typically stored on your own private registry but we will pull it from the NGINX hosted registry which requires authentication. Follow the [documentation](https://docs.nginx.com/nginx-ingress-controller/installation/using-the-jwt-token-docker-secret/) to configure it.
+
+In the [values-plus.yaml](values-plus.yaml) there is a section for globalConfiguration which will tell NGINX to listen on non-standard ports- see the redis section in this and the extra services.
+
+You will need to add the NGINX Helm repository to use it:
+
+```
+helm repo add nginx-stable https://helm.nginx.com/stable
+helm repo update
+helm install plus nginx-stable/nginx-ingress --namespace nginx-ingress -f values-plus.yaml
+```
+Note that the values file expects that you have already created a secret named regcred in the nginx-ingress namespace.
```

Comment:
You may want to go ahead and show how to create the "regcred" secret here.

Reply:
agree, thanks for reviewing!